tests/samples/distro/xmls/suggestions.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<components version="0.8" origin="jessie">
+  <component type="desktop">
+    <id>test.desktop</id>
```

Comment:
Merge components need some component to merge their data into - otherwise they don't exist in the pool.
Since there is no test.desktop, this would fail.